### PR TITLE
 Showcase of execution token distribution 

### PR DESCRIPTION
**Pull request by krhubert**
dependant on https://github.com/mesg-foundation/engine/pull/1650

Notes:

- i've send the coins when create execution, but it could be done when update (it is just a showcase)
- there can be only one runner owner - we should think of multiple owners.
- I guess there is no validation for execution create - all hash could be empty

### CODE REVIEW

File: cosmos/reciver/genesis.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package reciver
```
Comment (NicolasMahe) [inline cosmos/reciver/genesis.go]:
```suggestion
package receiver
```

File: x/execution/internal/keeper/keeper.go [hunk 2]
```diff
@@ -89,6 +96,57 @@ func (k *Keeper) Create(ctx sdk.Context, msg types.MsgCreateExecution) (*executi
...
+			Denom:  msg.Price[i].Denom,
+			Amount: sdk.NewIntFromBigInt(p.Div(p, big.NewInt(10))),
+		}
+		msg.Price[i].Sub(coin)
+
+		runnerPrice = append(runnerPrice, msg.Price[i])
+		servicePrice = append(servicePrice, coin)
+	}
+
+	if err := k.bankKeeper.SendCoins(ctx, msg.Signer, reciver.Reciver, runnerPrice); err != nil {
```
Comment (NicolasMahe) [inline x/execution/internal/keeper/keeper.go]: who is `reciver.Reciver`?

**Comment by NicolasMahe** [comment]
the transfer is correctly done:
localhost_3000_txs_DE37368B71EAA19A787F84948E659AE490EA5B89E199BFC350CB1B6AABB805E6(iPad)

```
➜  engine git:(feature/execution-price) ./bin/mesg-cli query account mesgtest1gzr4f4psys5ezcwcelp3sy855scasglch507c7 -o text
|
  address: mesgtest1gzr4f4psys5ezcwcelp3sy855scasglch507c7
  coins:
  - denom: atto
    amount: "20"
  public_key: ""
  account_number: 5
  sequence: 0
```

**Comment by antho1404** [comment]
didn't we say that we should attach a balance to the service and transfer to this balance and not directly the user

**Comment by NicolasMahe** [comment]
> didn't we say that we should attach a balance to the service and transfer to this balance and not directly the user

From the description:
> i've send the coins when create execution, but it could be done when update (it is just a showcase)

**Comment by NicolasMahe** [comment]
@krhubert you forget to close this PR:
from the forum:
> close https://github.com/mesg-foundation/engine/pull/1666 2 in favor of #1670

**Pull request closed without merging**